### PR TITLE
Home Page functionality part-2

### PR DESCRIPTION
**Pull request by OleksandraZaiets**

### CODE REVIEW

File: pages/HomePage.ts [hunk 1]
```diff
@@ -97,5 +101,45 @@ export class HomePage {
         await this.ccLink.click();
         await expect(this.page.url()).toContain(this.ccUrl);
     }
+
+        async verifyInformationTitles(): Promise<void> {
+            const urls = [
```
Comment (plakhotnikMykola) [inline pages/HomePage.ts:106]: It should be changed. Create constants.ts and add those cons as "export const URL"

File: pages/HomePage.ts [hunk 2]
```diff
@@ -37,6 +39,8 @@ export class HomePage {
         this.ccLink = page.getByRole(`link`, { name: `CC 4.0 BY-NC` });
         this.pngUrl = `/image`;
         this.ccUrl = `/licenses`;
+        this.informationTitles = page.locator(`#overview .row .col-sm`);
```
Comment (plakhotnikMykola) [inline pages/HomePage.ts:42]: Why it located here? All locator should be in fields

File: pages/HomePage.ts [hunk 3]
```diff
@@ -97,5 +101,45 @@ export class HomePage {
...
+            const container = await this.informationTitles.count();
+            for (let i = 0; i < container; i++) {
+                await expect(this.informationTitles.nth(i)).toBeVisible();
+                const textContent = await this.informationTitles.nth(i).textContent();
+                if (!textContent) continue;
+
+                if (i < urls.length) {
+                    await this.containerTitles.nth(i).click();
+
+                    if (i === 3) {
```
Comment (plakhotnikMykola) [inline pages/HomePage.ts:136]: What it is and why we do need it?

File: pages/HomePage.ts [hunk 4]
```diff
@@ -97,5 +101,45 @@ export class HomePage {
...
+                        await this.page.waitForTimeout(5000);
```
Comment (plakhotnikMykola) [inline pages/HomePage.ts:137]: Try to avoid hardcode timeout

**Review by plakhotnikMykola — CHANGES_REQUESTED** [review]

File: pages/HomePage.ts (hunk 1 above)
Comment (OleksandraZaiets) [reply pages/HomePage.ts:96]: Done

File: pages/HomePage.ts [hunk 5]
```diff
@@ -18,6 +18,8 @@ export class HomePage {
     readonly ccLink: Locator;
     readonly pngUrl: string;
     readonly ccUrl: string;
+    readonly informationTitles: Locator;
+    readonly containerTitles: Locator;
 
     constructor (page: Page, context: BrowserContext) {
```
[the remark this answers is not in the document]
Comment (OleksandraZaiets) [reply pages/HomePage.ts]: Done

File: pages/HomePage.ts (hunk 2 above)
Comment (OleksandraZaiets) [reply pages/HomePage.ts:31]: Fixed

File: pages/HomePage.ts (hunk 4 above)
Comment (OleksandraZaiets) [reply pages/HomePage.ts]: Fixed

File: pages/HomePage.ts (hunk 3 above)
Comment (OleksandraZaiets) [reply pages/HomePage.ts:113]: Found out

**Pull request merged**